lib/pdk/generators/module.rb

```diff
@@ -68,7 +68,7 @@ def self.invoke(opts = {})
           end
         end
 
-        PDK.answers.update!('template-url' => template_url)
+        PDK.answers.update!('template-url' => template_url) unless template_url == default_template_url
```

Comment:
as implemented this change also means that there is no way to reset to using the default without deleting/editing the ansers.json file.